OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -866,9 +940,10 @@ function New-Tentacle {
         [ValidateSet("Listen", "Poll")]
         [string]$communicationMode = "Listen",
         [int]$serverPort = 10943,
-        [ValidateSet("PublicIp", "FQDN", "ComputerName", "Custom")]
+        [ValidateSet("PublicIp", "FQDN", "ComputerName", "DHCP","Custom")]
         [string]$publicHostNameConfiguration = "PublicIp",
         [string]$customPublicHostName,
+        [string]$CidrRange,
```

Comment:
We may have forgotten to pass this parameter into `Add-TentacleToWorkerPool` and `Register-Tentacle` usage in the method?